misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java

```diff
@@ -248,7 +249,7 @@
  *
  * @since 2.0.0
  */
-public class NewFormattingTreeModifier extends FormattingTreeModifier {
+public class NewFormattingTreeModifier extends TreeModifier {
```

Comment:
Shall we rename this to `FormattingTreeModifier`?

Reply:
Since some fixes are still in progress, shall we rename this in a later PR?

Reply:
Also, initially, I sent the PR with this renaming [1], but the `FormattingTreeModifier` class seemed a little difficult to review here with both changes. WDYT?

[1] https://github.com/ballerina-platform/ballerina-lang/pull/26375/commits/0fdb82d649b01c657c96d709524392d52cc107e1

Reply:
ack. Let's rename it separately then.